codegen/templates/module_initializer.tmpl

```diff
@@ -42,14 +42,18 @@ func InitializeDependencies(
 	{{- else}}
 	tree := &DependenciesTree{}
 
-	{{- range $idx, $className := $instance.DependencyOrder}}
+	{{range $idx, $className := $instance.DependencyOrder}}
 	{{- $moduleInstances := (index $instance.RecursiveDependencies $className)}}
 	initialized{{$className | pascal}}Dependencies := &{{$className | title}}DependenciesNodes{}
 	tree.{{$className | title}} = initialized{{$className | pascal}}Dependencies
 
 	{{- range $idx, $dependency := $moduleInstances}}
 	{{- if $dependency.HasDependencies}}
```

Comment:
this `HasDependencies` check nolonger works because "everyone" has "default dependendices"

I think you want to get rid of this if statement all together.